test/unit/components/__snapshots__/button.test.jsx.snap

```diff
@@ -3,11 +3,9 @@
 exports[`ButtonComponent matches snapshot 1`] = `
 <span
   className=""
-  onClick={[Function]}
+  onClick={[MockFunction]}
   role="button"
 >
-  <div
-    className={undefined}
-  />
+  <div />
```

Comment:
Snapshots in newer versions of Jest no longer list `undefined` properties and make a distinction between a mocked function vs. a non-mocked function.